svgdigitizer/electrochemistry/cv.py

```diff
@@ -18,111 +18,166 @@
 #
 #  You should have received a copy of the GNU General Public License
 #  along with svgdigitizer. If not, see <https://www.gnu.org/licenses/>.
-#*********************************************************************
+# ********************************************************************
+from functools import cache
 import pandas as pd
 import matplotlib.pyplot as plt
 from pathlib import Path
+from astropy import units as u
+
+# The keys in this dict are unit strings that are identical to those
+# obtained by using str() on an astropy unit of a quantity, i.e.,
+# >>> from astropy import units as u
+# >>> q = 10 * u.A / u.cm**2
+# >>> str(q.unit)
+# 'A / cm2'
+# These string can directly be converted to a unit
+# >>> u.Unit('uA / cm2')
+unit_typos = {'uA / cm2': ['uA / cm2', 'uA / cm²', 'µA / cm²', 'µA cm⁻²', 'uA cm-2', 'uA / cm2'],
+              'A / cm2': ['A / cm2', 'A cm⁻²', 'A cm-2', 'A / cm2'],
+              'A': ['A', 'ampere', 'amps', 'amp'],
+              'mV': ['milliV', 'millivolt', 'milivolt', 'miliv', 'mV'],
+              'V': ['V', 'v', 'Volt', 'volt'],
+              'V / s': ['V s-1', 'V / s'],
+              'mV / s': ['mV / s', 'mV s-1', 'mV/s']}
 
 
 class CV():
-    def __init__(self, metadata, svgplot):
+    def __init__(self, svgplot, metadata=None):
         """
         metadata: dict
```

Comment:
That's a lie now because `metadata` can be `None` apparently. It's best to delete this comment altogether.

Reply:
done